src/SonarQube.TeamBuild.PreProcessor/SonarWebService.cs

```diff
@@ -139,8 +139,8 @@ public IList<ActiveRule> GetActiveRules(string qprofile)
                 {
                     var contents = downloader.Download(ws);
                     var json = JObject.Parse(contents);
-                    total = Convert.ToInt32(json["total"]);
-                    fetched += Convert.ToInt32(json["ps"]);
+                    total = json["total"].ToObject<int>();
+                    fetched += json["ps"].ToObject<int>();
```

Comment:
Why is that?

Reply:
Looks like a breaking change in the library - the old code was throwing exceptions.